src/lib/stores.ts

```diff
@@ -17,3 +17,4 @@ export const view = writable<string>('multi-day');
 export const viewedDate = writable<Date>(new Date());
 export const viewedMonth = writable<Date>(new Date());
 export const viewMode = writable<string>('normal');
+export const stateLoaded = writable<boolean>(false);
```

Comment:
do we need this new store or just reuse existing one, like loginState?

Reply:
Using `loginState == 'in'` in the code's current state does not work because `loginState` is set to `'in'` within the `initializeUserSessionData` function, which returns before `$settings` is assigned.  I spent a bit of time trying to rewrite the code so that `loginState` could be set to `'in'` after '$settings' is initialized, but I wasn't immediately able to find a straightforward to do that.  

And I suspect it might be desirable to keep  "app state is loaded" and "user is logged in" as separate conditions, although I can't state exactly why.

Reply:
maybe we should set this up to grab only the settings if session exists. ill try to review deeper later. but if this fixes stuff should be ok for now.